go/vt/vttls/vttls.go

```diff
@@ -61,6 +61,12 @@ func (mode *SslMode) String() string {
 	return string(*mode)
 }
 
+// Type returns the value type, part of the pflag Value interface
+// for allowing this to be used as a generic flag.
+func (mode *SslMode) Type() string {
+	return "SslMode"
```

Comment:
imo this should return "string" but idk what the best-practice for pflag is here see for example how `StringSliceVar` shows up as "strings" in the help output

Reply:
That's what I thought too, that's what I started with but then had failures.

Reply:
no no, i mean the _value_ returned by this function should be the literal string `"string"`, not the string `"SslMode"`

Reply:
That's what I thought you meant. It's also what I meant. 😄

Reply:
oh weird! TIL